Classes/ViewHelpers/Format/ICalendarDescriptionViewHelper.php

```diff
@@ -52,10 +52,10 @@ public function render(): string
         // Strip new lines
         $tmpDescription = str_replace(chr(10), '', $tmpDescription);
         // Glue everything together, so every line is max 75 chars
-        if (strlen($tmpDescription) > 75) {
-            $newDescription = substr($tmpDescription, 0, 63) . chr(10);
-            $tmpDescription = substr($tmpDescription, 63);
-            $arrPieces = str_split($tmpDescription, 74);
+        if (mb_strlen($tmpDescription) > 63) {
```

Comment:
Switching to `mb_strlen` for correct counting and max chars 63 for first line - like Google Calendar.